Controller/SubscriptionWallController.php

```diff
@@ -86,76 +82,22 @@ public function indexAction()
         $walls = $this->em
             ->getRepository('Integrated\Bundle\SubscriptionBundle\Model\SubscriptionWall')
             ->findAll();
-
+        
         return $this->templating->renderResponse('IntegratedSubscriptionBundle:SubscriptionWall:index.html.twig', [
             'walls' => $walls
         ]);
     }
 
     /**
-     * Creates a wall
-     *
-     * @return \Symfony\Component\HttpFoundation\RedirectResponse|\Symfony\Component\HttpFoundation\Response
-     */
```

Comment:
I don't think this should be deleted? Perhaps merge the master in your branch first?